packages/devtools-styleguide/src/components/Variables.js

```diff
@@ -1,22 +1,18 @@
 import React from "react";
-import PropTypes from "prop-types";
-const dom = React.DOM;
+const { Component } = React;
+import dom from "react-dom-factories";
+const { div, h2 } = dom;
 import postcss from "postcss";
 
 // require("./Variables.css");
 
 function iterate(list, cbk) {
-  var index = list.length;
-  for (var i = 0; i < index; i++) {
+  let index = list.length;
+  for (let i = 0; i < index; i++) {
     cbk(list[i]);
   }
 }
 
-function map(list, cbk) {
```

Comment:
why was this removed ?

Reply:
Because it is not used anywhere... we can put it back if you prefer.